__test__/shop.test.js

```diff
@@ -0,0 +1,89 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
+
+describe("Gilded Rose", function() {
+  it('Name does not change after update', () => {
```

Comment:
Is this the same test as the test in items.test.js on line 5?

Reply:
I think Tony mentioned that I shouldn't really even write tests for Item. I may just remove Item tests altogether.

Reply:
I talked to Andrew about this today. My reasoning for it is that Item is really just an object that holds data and it's one of those cases where people may skip out on testing just because it's pretty evident what it does. I think Andrew asked for the tests as doing pure TDD would require tests for every code you write.

I think it's a bit hard to distinguish which tests to write / not write when you're starting off, so I'd actually agree with Andrew and say just write all the tests for now :)